src/Symfony/Component/Validator/Constraints/Traverse.php

```diff
@@ -26,38 +25,16 @@ class Traverse extends Constraint
     public bool $traverse = true;
 
     /**
-     * @param bool|array<string,mixed>|null $traverse Whether to traverse the given object or not (defaults to true). Pass an associative array to configure the constraint's options (e.g. payload).
```

Comment:
could be worth doing this on 7.4: docblocks should be only about the non-deprecated API